src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,80 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final int INDEX_OF_SUPPLY = 0;
+    private static final int INDEX_OF_BUY = 1;
+    private static final int INDEX_OF_RESULT = 2;
+    private static final int INDEX_OF_SUPPLY_OR_BUY = 0;
+    private static final int INDEX_OF_QUANTITY = 1;
+    private static final int REQUIRED_CAPACITY = 3;
 
     public void getStatistic(String fromFileName, String toFileName) {
+        int[] supplyAndBuyAndResult = getSupplyAndBuyAndResult(fromFileName);
+        createReport(supplyAndBuyAndResult, toFileName);
+    }
+
+    private int[] getSupplyAndBuyAndResult(String fromFileName) {
```

Comment:
there should be 3 private methods to properly divide business logic: `readFromFile`, `createReport` and `writeToFile`

Reply:
done